modules/labs-react/drawer/lib/DrawerHeader.tsx

```diff
@@ -88,9 +81,15 @@ export class DrawerHeader extends React.Component<DrawerHeaderProps, {}> {
 
     return (
       <HeaderContainer borderColor={borderColor} {...elemProps} headerColor={headerColor}>
-        <HeaderTitle id={id} inverse={inverse} title={title}>
+        <Heading
+          size="small"
+          title={title}
+          fontWeight="bold"
+          color={inverse ? 'inverse' : typeColors.heading}
+          paddingRight="xxxs"
+        >
```

Comment:
do we need a styled component here? I think you move all the styles and just use `TypeBodyLevel` or the `TypeHeadingLevel` ?
```
<TypeHeadingLevel whitespace="nowrap" textOverflow="ellipsis" overflow="hidden" size="small" as="h4" title={title} fontWeight="bold" color={inverse ? 'inverse' : typeColors.heading} padding="xxxs">{title}</TypeHeadingLevel>
```

Reply:
nice catch!